### PR TITLE
Add postgres cdc to vertx

### PR DESCRIPTION
**Pull request by henneberger**

### CODE REVIEW

**Comment by henneberger** [comment]
Let's open a ticket for subscriptions

File: profiles/default/.env.ftl [hunk 1]
```diff
@@ -1,4 +1,10 @@
-COMPOSE_FILE=${config["enabled-engines"]?map(engine -> engine + ".compose.yml")?join(":")}
+<#assign composeFiles = "">
+<#list config["enabled-engines"] as engine>
+  <#if engine != "postgres_log">
```
Comment (henneberger) [inline profiles/default/.env.ftl:3]: Use an empty .compose.yaml like the other engines.

File: sqrl-planner/src/main/java/com/datasqrl/engine/database/relational/ddl/statements/InsertStatement.java [hunk 2]
```diff
@@ -16,6 +17,7 @@
 import org.apache.calcite.sql.SqlNodeList;
 import org.apache.calcite.sql.parser.SqlParserPos;
 
+@JsonPropertyOrder({"tableName", "sql", "params"})
```
Comment (henneberger) [inline sqrl-planner/src/main/java/com/datasqrl/engine/database/relational/ddl/statements/InsertStatement.java:20]: nice

**Review by nandorsoma — APPROVED** [review]

**Comment by nandorsoma** [comment]
Closes #732

**Pull request merged**